libs/core/src/lib/combobox/combobox.component.ts

```diff
@@ -483,21 +537,38 @@ export class ComboboxComponent implements ControlValueAccessor, OnInit, OnChange
         }
     }
 
-    private _handleClickActions(term): void {
+
+    /** @hidden*/
+    private _popoverOpenHandle(): void {
+        if (this._hasDisplayedValues) {
```

Comment:
Focus trap is based on whether component has `displayedValues`?

Reply:
In combobox, the popover body is shown only, when there are displayed values. That's why I base focus trap on fact if it's open + if there are some values shown.